### PR TITLE
Various bug fixes and removing useless modules

### PR DESCRIPTION
**Pull request by czho**
**Describe the pull**
Fix issues with tooltip, Add shadow option to CustomFont, Add ElytraFix, remove antibot, Remove EndTeleport

**Additional context**
Things from issue #171

### CODE REVIEW

File: src/main/kotlin/com/lambda/client/module/modules/combat/AutoMend.kt [hunk 1]
```diff
@@ -206,7 +206,6 @@ object AutoMend : Module(
         for (entity in mc.world.loadedEntityList) {
             if (entity !is EntityPlayer) continue
             if (entity.isFakeOrSelf) continue
-            if (AntiBot.isBot(entity)) continue
             if (mc.player.getDistance(entity) > pauseNearbyRadius) continue
             if (FriendManager.isFriend(entity.name)) continue
```
Comment (d1snin) [inline src/main/kotlin/com/lambda/client/module/modules/combat/AutoMend.kt:210]: this is a pure shit. Use `||` or extract an extension function for it.

File: src/main/kotlin/com/lambda/client/module/modules/misc/ElytraFix.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    init {
+        safeListener<PacketEvent.Receive> {
+            if (it.packet is SPacketPlayerPosLook && player.isElytraFlying) {
+                world.getLoadedEntityList().filterIsInstance<EntityFireworkRocket>().forEach {
+                    world.removeEntity(it)
+                }
+            }
+        }
+    }
+}
```
Comment (d1snin) [inline src/main/kotlin/com/lambda/client/module/modules/misc/ElytraFix.kt:25]: afaik you can just add an option to the main elytraflight module

Comment (czho) [reply src/main/kotlin/com/lambda/client/module/modules/misc/ElytraFix.kt:25]: yea but not everyone uses elytraflight, this is a fix for using rockets with elytra

File: src/main/kotlin/com/lambda/client/module/modules/combat/AutoMend.kt (hunk 1 above)
Comment (czho) [reply src/main/kotlin/com/lambda/client/module/modules/combat/AutoMend.kt:210]: true, yet again, ask the kami blue devs why they did this, ill fix that

Comment (d1snin) [reply src/main/kotlin/com/lambda/client/module/modules/combat/AutoMend.kt:210]: there is literally no difference between using normal condition expressions and this horrible one

File: src/main/kotlin/com/lambda/client/module/modules/misc/ElytraFix.kt (hunk 2 above)
Comment (d1snin) [reply src/main/kotlin/com/lambda/client/module/modules/misc/ElytraFix.kt:25]: ah, ok, then it makes some sense

**Comment by czho** [comment]
PR is fucked ima redo it real quick

File: src/main/kotlin/com/lambda/client/module/modules/misc/ElytraFix.kt (hunk 2 above)
Comment (Avanatiker) [reply src/main/kotlin/com/lambda/client/module/modules/misc/ElytraFix.kt:25]: What if you wanna use the fix but only fly vanilla?

**Pull request closed without merging**